homeassistant/components/digital_ocean/__init__.py

```diff
@@ -55,8 +59,12 @@ def setup(hass: HomeAssistant, config: ConfigType) -> bool:
         _LOGGER.error("API token not valid for authentication")
         return False
 
-    hass.data[DATA_DIGITAL_OCEAN] = digital
-
+    hass.data[HASS_DATA_DIGITAL_OCEAN] = digital
```

Comment:
Our standard nowadays is to use the integration domain instead.

```suggestion
    hass.data[DOMAIN] = digital
```

Reply:
I just moved/renamed the constant, and wanted to keep the same logical behavior.
I can change it if you deem important.

Reply:
Reverted